diff -ur kernel-source.org/arch/arm64/boot/dts/freescale/imx8mm-evk.dtsi kernel-source/arch/arm64/boot/dts/freescale/imx8mm-evk.dtsi
--- kernel-source.org/arch/arm64/boot/dts/freescale/imx8mm-evk.dtsi	2023-09-20 14:08:41.822947549 +0100
+++ kernel-source/arch/arm64/boot/dts/freescale/imx8mm-evk.dtsi	2023-09-20 16:08:00.194748554 +0100
@@ -130,6 +130,7 @@
 		audio-cpu = <&sai1>;
 		audio-codec = <&ak4458_1>, <&ak4458_2>;
 		reset-gpios = <&pca6416 4 GPIO_ACTIVE_LOW>;
+		status = "disabled";
 	};
 
 	sound-ak5558 {
@@ -409,6 +410,7 @@
 		reg = <0x10>;
 		AVDD-supply = <&reg_audio_board>;
 		DVDD-supply = <&reg_audio_board>;
+		status = "disabled";
 	};
 
 	ak4458_2: ak4458@12 {
@@ -416,6 +418,7 @@
 		reg = <0x12>;
 		AVDD-supply = <&reg_audio_board>;
 		DVDD-supply = <&reg_audio_board>;
+		status = "disabled";
 	};
 
 	ak5558: ak5558@13 {
@@ -424,6 +427,7 @@
 		reset-gpios = <&pca6416 3 GPIO_ACTIVE_LOW>;
 		AVDD-supply = <&reg_audio_board>;
 		DVDD-supply = <&reg_audio_board>;
+		status = "disabled";
 	};
 
 	ak4497: ak4497@11 {
@@ -433,6 +437,7 @@
 		AVDD-supply = <&reg_audio_board>;
 		DVDD-supply = <&reg_audio_board>;
 		dsd-path = <1>;
+		status = "disabled";
 	};
 
 	ov5640_mipi: ov5640_mipi@3c {
