diff -ur git.org/include/configs/imx8mm_evk.h git/include/configs/imx8mm_evk.h
--- git.org/include/configs/imx8mm_evk.h	2023-09-19 17:04:45.921296582 +0100
+++ git/include/configs/imx8mm_evk.h	2023-09-19 17:06:34.504395138 +0100
@@ -138,8 +138,8 @@
 	"image=Image\0" \
 	"splashimage=0x50000000\0" \
 	"console=ttymxc1,115200\0" \
-	"fdt_addr_r=0x43000000\0"			\
-	"fdt_addr=0x43000000\0"			\
+	"fdt_addr_r=0x44000000\0"			\
+	"fdt_addr=0x44000000\0"			\
 	"fdt_high=0xffffffffffffffff\0"		\
 	"boot_fit=no\0" \
 	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0" \
